ui-manchette-with-spacetimechart/src/helpers.ts

```diff
@@ -1,7 +1,7 @@
 import type {
-  OperationalPointType,
+  InteractiveWaypoint,
   ProjectPathTrainResult,
-  StyledOperationalPointType,
+  Waypoint,
 } from '@osrd-project/ui-manchette/dist/types';
 import type {
   OperationalPoint,
```

Comment:
Concerns the spacetime part but this should be renamed too ?

Reply:
I think we should rename it yes, but maybe later ?

Reply:
Sure